### DISCUSSION TITLE
my self host runner cann’t delete , http request 401

### DISCUSSION THREAD

**Discussion by Nick-0314** [community/community#36639] [Actions]
![image](https://user-images.githubusercontent.com/50027289/196677143-5089d527-f2ff-498d-b015-418d0e895ab5.png)

**Comment by ossanna16** [comment]
Hey there @mytting and thank you for asking a good question! I'm glad you found our community 🙂

**Reply by Nick-0314** [reply]
> Hey there @mytting and thank you for asking a good question! I'm glad you found our community 🙂

I use Safari browser is normal, and successfully removed, Chrome may have some problems

**Comment by alex88** [comment]
Same here, tried both chrome and firefox, both get a 401 trying to load the delete runner modal

**Reply by ChristopherHX** [reply]
You can use the runner registration token from the runner registration page to remove your runner, it should have the same permissions as the remove token. However this might change at any time with a new permission system.

**EDIT I get the same 401 error Microsoft Edge on windows and the remove dialog loads infinitely**

**Reply by alex88** [reply]
Oh ok so I'm not alone :D

**Comment by mgax** [comment]
I can confirm, not possible to remove a runner from Firefox, while Safari works.
